notebooks/README.md

```diff
@@ -1,14 +1,26 @@
 # Lumigator Demo Notebooks
-A collection of Jupyter notebooks and scripts showcasing Lumigator's functionalities. For installing Lumigator, see the main [README.md](README.md)
 
-The notebook runs on Jupyter. If you don't yet have Jupyter set up: 
+A collection of Jupyter notebooks and scripts showcasing Lumigator's functionalities. For installing
+Lumigator, see the main [README.md](README.md)
 
-1. create a new virtualenv for lumigator
-2. `cd notebooks` (assuming you were in the root directory of the repo)
-3. `pip install -r requirements.txt`
-4. `pip install jupyterlab` 
-5. set up the environment so that both our backend and our ray cluster point to localhost and start jupyterlab:
-```LUMIGATOR_SERVICE_HOST="localhost" RAYCLUSTER_KUBERAY_HEAD_SVC_PORT_8265_TCP_ADDR="localhost" jupyter-lab```
+The notebook runs on Jupyter. If you don't yet have Jupyter set up:
 
-A new browser window will open pointing at your new jupyterlab. From there, open the `walkthrough.ipynb` file.
+1. Create a new virtualenv for lumigator: `python3 -m venv <path-to-virtualenv>`.
```

Comment:
Let's use `uv` here for venv creation to keep consistent with the rest of the app

Reply:
@veekaybee my assumption is that the user may not have `uv` installed. But I can add it, for sure. WDYT?

Reply:
yep, would be helpful to specify both

Reply:
I added both commands, how you can use `uv` to create a virtual environment and how you can do it if you don't have `uv` installed.